tests/integration_test/CMakeLists.txt

```diff
@@ -18,16 +18,17 @@ afr_module_dependencies(
 )
 
 # FreeRTOS+TCP Test
-afr_test_module(freertos_tcp_test)
+if(TARGET AFR::freertos_plus_tcp)
```

Comment:
Should the core_mqtt_integration test module above also be wrapped in an if like this one?

Reply:
It is not required for the `core_mqtt_integration` test module as _every_ board supports the MQTT library. It is needed for the `freertos_plus_tcp` module as some boards do not support the module..

Reply:
ok thanks